src/app.md

```diff
@@ -193,6 +194,63 @@ fn run() -> Result<()> {
 # quick_main!(run);
 ```
 
+[ex-tar-strip-prefix]: #ex-tar-strip-prefix
+<a name="ex-tar-strip-prefix"></a>
+## Decompress a tarball while removing a prefix from the paths
+
+[![flate2-badge]][flate2] [![tar-badge]][tar] [![cat-compression-badge]][cat-compression]
+
+Strip a path prefix from the entries of a tarball before unpacking them.
+
+In this case we have a tarball, `archive.tar.gz`, whose contents are inside the
+sub-directory `bundle/logs`. First we create a [`tar::Archive`], similar to the
+[decompress a tarball](#ex-tar-decompress) example, using [`GzDecoder`]. Then we
+iterate over the [`Archive::Entries`], using [`Path::strip_prefix`] to remove
```

Comment:
`Archive::Entries` is a struct but points to method producing this struct. please either change the link text or url